styles/prosilver/template/event/index_body_markforums_before.html

```diff
@@ -1,3 +1,6 @@
+<!-- IF S_RECENT_TOPICS && S_LOCATION_SIDE -->
+<!-- ELSE -->
 {% if MCHAT_INDEX and MCHAT_LOCATION %}
 	{% INCLUDE '@dmzx_mchat/mchat_body.html' %}
```

Comment:
Please add your two `S_`switches to the existing ìf` condition. Also, please don't mix old phpBB template syntax with Twig syntax and stick with Twig syntax only.